components/react-live-markdown.js

```diff
@@ -0,0 +1,58 @@
+import React, {Component} from 'react'
+import axios from 'axios'
+import remark from 'remark'
+import remarkHtml from 'remark-html'
+import visit from 'unist-util-visit'
+import CodePreview from './code-preview'
+
+class ReactLiveMarkdown extends Component {
+  constructor() {
+    super()
+    this.state = {markdownComponents: <div>loading</div>}
+  }
+  componentDidMount() {
+    if (typeof window === 'undefined' || window.__NEXT_DATA__ === undefined) {
```

Comment:
Nope. I copy/pasted this. Still new to next :) Glad you're around! You've been awesome!